src/languages/es.json

```diff
@@ -0,0 +1,118 @@
+        },
+        "error": {
+            "fail_create_account": "Fallo al crear la cuenta."
+        },
+        "test_mode": "Esto es un<br><strong>Monedero web de TEST!</strong>",
+    },
+    "login_view": {
+        "header": "Introduce tu semilla",
+        "guide_line_1": "Por favor, introduce tu semilla secreta para acceder a tu cuenta.",
+        "guide_line_2": "Por favor, guarda la semilla secreta de forma segura, de lo contrario la seguridad de tu monedero podría ser comprometida.",
+        "error": {
+            "invalid_secret_seed": "Semilla secreta inválida o balance insuficiente (1 BOScoin mínimo)."
+        },
+    },
+    "send_coin": {
+        "input_recipient_address": "Introduce la dirección pública del destinatario",
+        "input_amount": "Introduce la cantidad que deseas enviar",
+        "transaction_fee": "Comisión de la transacción",
+        "total_will_be_sent": "Un total de {amount} BOScoins serán enviados al destinatario",
+        "error": {
+            "public_address_null": "Introduce la dirección pública del destinatario",
+            "transaction_amount_null": "Introduce la cantidad para la transacción",
+            "transaction_amount_zero": "Introduce la cantidad para la transacción",
+            "not_enough_balance": "Balance Insuficiente",
+            "incorrect_public_address": "Dirección pública incorrecta",
+            "minimum_balance": "El balance mínimo tras la transacción ha de ser de almenos {minimum_balance} BOS.",
+            "decimal_limit": "Solo puedes introducir 7 decimales.",
+        },
+    },
+    "key_generator": {
+        "header": "Pulsa para generar una nueva pareja de claves",
+        "description_line1": "Esta herramienta no te creará una cuenta.",
+        "description_line2": "Deberás mandar una operación del tipo create_account a la siguiente",
+        "description_line3": "dirección con un balance inicial para que pueda existir. ",
+        "description_line4": "Guarda tu semilla de forma segura.",
+        "warn_line1": "Por facor, copia tu dirección pública y la semilla secreta y guardaló en algún lugar seguro.",
+        "warn_line2": "Asegúrate de no perder ni revelar a nadie tu Semilla Secreta.",
+        "warn_line3": "¡Puedes perder todas tus monedas!",
+    },
+    "record_seed": {
+        "warn_header1": "Guarda tus semillas",
+        "warn_header2_1": "Guarda tus semillas",
+        "warn_header2_2": " ¡AHORA!",
+        "warn_header3": "¡¡¡NO PIERDAS TUS SEMILLAS!!!",
+        "warn_text1_1": "Asegúrate de guardar tu Dirección Pública y la Semilla Secreta de forma segura.",
+        "warn_text1_2": "Asegúrate de no perder ni compartir tu Semilla Secreta. Es la llave de tu monedero.",
+        "warn_text2_1": "POR FAVOR, revisa doblemente que tienes las semillas correctas.",
+        "warn_text2_2": "You are solely responsible for the security of your Seeds.",
```

Comment:
Can you translate this sentence too?

"You are solely responsible for the security of your Seeds."